parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -274,7 +274,7 @@ sqliteCodebase debugName root localOrRemote action = do
             Sqlite.runTransaction conn do
               CodebaseOps.putRootBranch rootBranchCache (Branch.transform (Sqlite.unsafeIO . runInIO) branch1)
 
-        rootBranchUpdates :: MonadIO m => TVar (Maybe (Sqlite.DataVersion, a)) -> m (IO (), IO (Set Branch.Hash))
+        rootBranchUpdates :: MonadIO m => TVar (Maybe (Sqlite.DataVersion, a)) -> m (IO (), IO (Set Branch.CausalHash))
```

Comment:
BTW this function currently does nothing, but implies that it detects root branch updates;

Reply:
Yeah, we should dig in and decide whether to make it detect root branch updates, or delete it.